themes/bubble-wrap-icons-shields-usa.yaml

```diff
@@ -0,0 +1,584 @@
+            'US:TN-3char': [438, 170, 48, 38]
+            'US:TN-4char': [376, 170, 54, 38]
+            'US:TN-5char': [308, 170, 62, 38]
+            'US:US-1char': [422, 684, 44, 44]
+            'US:US:Business-1char': [422, 684, 44, 44]
+            'US:US:Truck-1char': [422, 684, 44, 44]
+            'US:US:Alternate-1char': [422, 684, 44, 44]
+            'US:US-2char': [472, 684, 52, 46]
+            'US:US:Business-2char': [472, 684, 52, 46]
+            'US:US:Truck-2char': [472, 684, 52, 46]
+            'US:US:Alternate-2char': [472, 684, 52, 46]
+            'US:US-3char': [346, 684, 68, 48]
+            'US:US:Business-3char': [346, 684, 68, 48]
+            'US:US:Truck-3char': [346, 684, 68, 48]
+            'US:US:Alternate-3char': [346, 684, 68, 48]
+            'US:US-4char': [266, 684, 76, 48]
+            'US:US:Business-4char': [266, 684, 76, 48]
+            'US:US:Truck-4char': [266, 684, 76, 48]
+            'US:US:Alternate-4char': [266, 684, 76, 48]
+            'US:US-5char': [178, 682, 82, 50]
+            'US:US:Business-5char': [178, 682, 82, 50]
+            'US:US:Truck-5char': [178, 682, 82, 50]
+            'US:US:Alternate-5char': [178, 682, 82, 50]
+            'US:UT-1char': [836, 376, 50, 44]
+            'US:UT-2char': [776, 376, 52, 44]
+            'US:UT-3char': [706, 376, 62, 44]
+            'US:UT-4char': [632, 376, 68, 44]
+            'US:UT-5char': [548, 376, 78, 46]
+            'US:VA-1char': [400, 478, 36, 42]
+            'US:VA-2char': [350, 478, 42, 42]
+            'US:VA-3char': [292, 478, 50, 42]
+            'US:VA-4char': [226, 478, 58, 42]
+            'US:VA-5char': [152, 478, 68, 42]
+            'US:VT-1char': [308, 328, 46, 34]
+            'US:VT-2char': [254, 328, 48, 34]
+            'US:VT-3char': [194, 328, 54, 34]
+            'US:VT-4char': [126, 328, 62, 34]
+            'US:VT-5char': [52, 328, 68, 34]
+            'US:WA-1char': [50, 532, 44, 44]
+            'US:WA-2char': [0, 532, 44, 44]
+            'US:WA-3char': [942, 478, 56, 46]
+            'US:WA-4char': [870, 478, 66, 46]
+            'US:WA-5char': [790, 478, 74, 46]
+            'US:WI-1char': [104, 478, 40, 40]
+            'US:WI-2char': [56, 478, 42, 42]
+            'US:WI-3char': [0, 478, 50, 42]
+            'US:WI-4char': [940, 430, 56, 40]
+            'US:WI-5char': [868, 430, 64, 40]
+            'US:WY-1char': [704, 274, 42, 42]
+            'US:WY-2char': [652, 272, 44, 44]
+            'US:WY-3char': [590, 274, 54, 40]
+            'US:WY-4char': [520, 274, 64, 40]
+            'US:WY-5char': [446, 274, 68, 40]
+            county_shield-1char: [572, 632, 36, 36]
+            county_shield-2char: [832, 632, 44, 36]
+            county_shield-3char: [774, 632, 52, 36]
+            county_shield-4char: [700, 632, 68, 36]
+            county_shield-5char: [614, 632, 80, 36]
+            
+layers:
+    roads:
+        shields:
+            shields-usa:
+                filter:
```

Comment:
No `all` needed in filter

Reply:
b38b8910848b62cf3804ad8b70ab04ba8a9d2beb